lib/fog/openstack/requests/compute/evacuate_server.rb

```diff
@@ -3,12 +3,12 @@ module Compute
     class OpenStack
       class Real
         def evacuate_server(server_id, host, on_shared_storage, admin_password = nil)
+          evacuate_hash = {}
```

Comment:
Works for me on the variable name change (I've seen this style used often in the past, but yeah, I'll fix it). I can change the signature too. I was trying to minimize external changes to what was essentially a bug fix here, but if there are no objections to making those all default to nil, that makes more sense to me anyway.